test/api_test.js

```diff
@@ -540,6 +540,40 @@ describe("KintoClient", () => {
     });
   });
 
+  /** @test {KintoClient#listPermissions} */
+  describe("#listPermissions()", () => {
+    const data = [{id: "a"}, {id: "b"}];
+
```

Comment:
Dunno as `capable` is pretty well tested already... ok why not, never too safe I guess :)